build-release-tools/application/jira_epic_report.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Copyright 2017, DELLEMC, Inc.
+from jira import JIRA
+import sys
+import argparse
+import common
+
+class JIRAOperator(object):
+    def __init__(self, server, username, password):
+        # Initial jira 
+        # set verify as false to get rid of requests.exceptions.SSLError: [Errno 1] _ssl.c:507: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed
+        options = {'server': server, 'verify': False}
+        self.__jira = JIRA(options, basic_auth=(username, password))
+        
+    def search_issues(self, sql_str):
+        issues = self.__jira.search_issues(sql_str)
+        return issues
+
+    def search_open_bugs_by_priority(self, project, priority):
+        JQL_str="project in ({0}) and issuetype = Bug and status not in (done) and priority = {1}".format(project, priority)
```

Comment:
I would suggest to make "done" as a default parameter value of this function.
in case  new status which similar to done(like "invalid" as another end-status) added in the future.